src/network/protocol/messages/lib.zig

```diff
@@ -60,13 +62,49 @@ pub const MessageTypes = enum {
     sendcmpct,
     feefilter,
     filterclear,
+    getdata,
     block,
     filteradd,
     notfound,
     sendheaders,
     filterload,
 };
 
+pub const InventoryItem = struct {
```

Comment:
Can you move it in it's own file please?

Using this zig feature: https://zig.news/gowind/zig-files-are-structs-288j

Reply:
same for all the other structs that have been added in this file

Reply:
maybe `messages/types/InventoryItem.zig`

Reply:
there is already `src/network/protocol/NetworkAddress.zig`.
Move it under /types/